Dockerfile

```diff
@@ -97,4 +97,7 @@ COPY --chown=node:node examples examples
 # This switches many Node.js dependencies to production mode.
 ENV NODE_ENV=production
 
+# This disables node snapshot for Node 20 to work with the Scaffolder
+ENV NODE_OPTIONS="--no-node-snapshot"
```

Comment:
Doc ref: https://backstage.io/docs/features/software-templates/#getting-started

Reply:
And here for the Dockerfile: https://backstage.io/docs/deployment/docker.